### PR TITLE
fix: dependencies

### PR DESCRIPTION
**Pull request by rflechtner**
Use the full sdk release instead of a release candidate; also take advantage of the api augmentation provided by the sdk since 0.29 instead of manually defining on-chain types.
Finally, remove a forced resolution of a babel dependency whose motivation is unclear.

### CODE REVIEW

File: src/index.tsx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import React from 'react'
 import ReactDOM from 'react-dom'
-import '@polkadot/api-augment'
+import '@kiltprotocol/augment-api'
```
Comment (weichweich) [inline src/index.tsx:3]: `@polkadot/api-augment` vs `@kiltprotocol/augment-api` did we swap the name here and broke the convention? 😂

**Review by weichweich — APPROVED** [review]
Nice love it!

File: src/index.tsx (hunk 1 above)
Comment (rflechtner) [reply src/index.tsx:3]: was there a convention? 😨

Comment (weichweich) [reply src/index.tsx:3]: Not a convention. but if polkadot names it `api-augment` i would consider it the established name.

**Comment by rflechtner** [comment]
> Were these changes required? If not, we should consider that we were told to only put effort into it if strictly necessary.

I think it's just a pretty bad look on a public repo to be using a release candidate for a live application. We have the full release out, no reason not to switch to it.

File: src/index.tsx (hunk 1 above)
Comment (rflechtner) [reply src/index.tsx:3]: welllllll too late

**Comment by rflechtner** [comment]
I'd also say these changes are maintenance; using our api augmentation just makes it easier to find breaking type changes and fix them if that ever becomes necessary.
And using the runtime call instead of state queries is simply a better example for anyone using the stakeboard as a reference implementation.

**Pull request merged**